make/testing_flags.mk

```diff
@@ -1,8 +1,14 @@
 DEV_TAGS = dev
+RPC_TAGS = chainrpc walletrpc signrpc invoicesrpc autopilotrpc watchtowerrpc
```

Comment:
IIRC there are a few other subservers also, should we just enable them?

Reply:
The original idea was to only include the subservers that are also present in the release build, but actually I think it makes sense to include all.
This also changes what we itest, but imho it is more correct to itest on the full build and have a smaller subset for release than itest on the smaller subset. PTAL